src/snowflake/snowpark/session.py

```diff
@@ -191,6 +191,38 @@ class Session:
     A :class:`Session` object is not thread-safe.
     """
 
+    class RuntimeConfig:
+        def __init__(self, session: "Session", conf: Dict[str, Any]) -> None:
+            self._session = session
+            for key, val in conf.items():
+                if hasattr(Session, key):
+                    setattr(session, key, val)
```

Comment:
hmmm it should have side effect right? Now people can use session.account to get the account, so it creates new properties exposed to public users, but not documented.

Reply:
`account` won't be added since it's not an existing attribute of `Session`, but I see how this could be dangerous: it could be used to overwrite any attributes of `Session`, I will change this to check `self.is_mutable` too.